src/gui/uithememanager.cpp

```diff
@@ -99,29 +123,30 @@ QIcon UIThemeManager::getIcon(const QString &iconId, const QString &fallback) co
 {
 #if (defined(Q_OS_UNIX) && !defined(Q_OS_MAC))
     if (m_useSystemTheme) {
-        QIcon icon = QIcon::fromTheme(iconId);
+        QIcon icon = QIcon::fromTheme(m_iconMap[iconId].toString());
         if (icon.name() != iconId)
```

Comment:
You should compare with mapped icon id to follow current logic. Otherwise it's meaningless.